### ISSUE TITLE
git extension: "never" option for "openAfterClone" extension configuration

### ISSUE THREAD

**Issue by sij1nk** [microsoft/vscode#228941]
Hi!

tl;dr: I would like to have a "never" option for the "openAfterClone" config of the built-in git extension. I would be willing to submit a PR for this

---

I am working on a vscode extension that, amongst other things, wants to let users clone git repositories, and if the repository passes certain checks (e.g. its folder structure is correct), then:

- execute some cli commands within it
- open it in the current window

Since [vscode's built-in git extension](https://github.com/microsoft/vscode/tree/f9d9605159d95771db8994af5fdf1c968095509f/extensions/git) already provides UI for asking for the repository url, parent path, SSH passphrase, etc., it makes sense for us to reuse it for our purpose, specifically, by executing the [git.clone](https://github.com/microsoft/vscode/blob/f9d9605159d95771db8994af5fdf1c968095509f/extensions/git/src/commands.ts#L983) command. However, we would like the command *not* to automatically open the repository after cloning. The extension exposes an [openAfterClone](https://github.com/microsoft/vscode/blob/f9d9605159d95771db8994af5fdf1c968095509f/extensions/git/src/commands.ts#L856) config, but none of the options is appropriate for us:

- "always", "alwaysNewWindow": not what we want
- "whenNoFolder": in our use case, there are no open workspace folders, so this behaves the same as "always"
- "prompt": prompts the user what to do; we do not want to do this

Ideally, we'd like to have a "never" option, that would tell the git extension to not do anything after the clone operation completes, so we can do our thing afterwards. We would set the "openAfterClone" to "never" before cloning, and then reset it to its initial value afterwards, in order not to mess with the user's settings if they have set it to something else.

---

The only issue I found that is sort of related is https://github.com/microsoft/vscode/issues/145776 but it does not solve my problem.

Thanks!
